src/commands/sourcemaps/resolve.rs

```diff
@@ -0,0 +1,107 @@
+use std::fs;
+use std::path::PathBuf;
+
+use anyhow::{format_err, Result};
+use clap::{Arg, ArgMatches, Command};
+use sourcemap::{DecodedMap, Token};
+
+use crate::utils::args::validate_int;
+
+pub fn make_command(command: Command) -> Command {
+    command
+        .about("Resolve sourcemap for a given line/column position.")
+        .arg(
+            Arg::new("path")
+                .value_name("PATH")
+                .help("The sourcemap to resolve."),
+        )
+        .arg(
+            Arg::new("line")
+                .long("line")
+                .short('l')
+                .value_name("LINE")
+                .validator(validate_int)
+                .help("Line number for minified source."),
+        )
+        .arg(
+            Arg::new("column")
+                .long("column")
+                .short('c')
+                .value_name("COLUMN")
+                .validator(validate_int)
+                .help("Column number for minified source."),
+        )
+}
+
+/// Retrurns the zero indexed position from matches
+fn lookup_pos(matches: &ArgMatches) -> Option<(u32, u32)> {
+    Some((
+        matches
+            .value_of("line")
+            .map_or(0, |x| x.parse::<u32>().unwrap() - 1),
+        matches
+            .value_of("column")
+            .map_or(0, |x| x.parse::<u32>().unwrap() - 1),
+    ))
+}
+
+fn print_token(token: &Token<'_>) {
+    if let Some(name) = token.get_name() {
+        println!("  name: {:?}", name);
+    } else {
+        println!("  name: not found");
+    }
+    if let Some(source) = token.get_source() {
+        println!("  source file: {:?}", source);
+    } else {
+        println!("  source file: not found");
+    }
+    println!("  source line: {}", token.get_src_line());
+    println!("  source column: {}", token.get_src_col());
+    println!("  minified line: {}", token.get_dst_line());
+    println!("  minified column: {}", token.get_dst_col());
```

Comment:
what's with the spaces at the start of lines here?

Reply:
It was meant to make the distinction between the lookup input and output more clear:

<img width="524" alt="image" src="https://user-images.githubusercontent.com/1523305/162755668-285e35fd-7f14-461d-9638-619a8c62f90e.png">

Reply:
Also added better indent formatting (code ugly af, but it works :p)

<img width="462" alt="image" src="https://user-images.githubusercontent.com/1523305/162758883-d2ff44ba-7114-4139-8d75-d8f24ba88c05.png">